apps/omg_watcher_info/lib/omg_watcher_info/release_tasks/init_postgresql_db.ex

```diff
@@ -13,59 +13,21 @@
 # limitations under the License.
 
 defmodule OMG.WatcherInfo.ReleaseTasks.InitPostgresqlDB do
-  @moduledoc """
-  A release task that performs database initialization, namely database creation and migration.
+  @moduledoc false
+  @app :omg_watcher_info
 
-  This release task can also be run on initialized database in order to migrate up to the latest
-  database schema.
-  """
-  alias Ecto.Migrator
-  alias OMG.Utils.CLI
-  @start_apps [:crypto, :ssl, :postgrex, :phoenix_ecto, :ecto_sql, :telemetry]
-  @apps [:omg_watcher_info]
-
-  def run() do
-    Enum.each(@start_apps, &Application.ensure_all_started/1)
-    Enum.each(@apps, &init_pg_db/1)
-    :init.stop()
-  end
-
-  defp init_pg_db(app_name) do
-    _ = Application.load(app_name)
-    repos = Application.get_env(app_name, :ecto_repos, [])
-
-    Enum.each(repos, &run_create_for/1)
-    Enum.each(repos, & &1.start_link(pool_size: 2))
-    Enum.each(repos, &run_migrations_for/1)
-  end
-
-  defp run_create_for(repo) do
-    case repo.__adapter__.storage_up(repo.config) do
-      :ok ->
-        CLI.info("The database for #{inspect(repo)} has been created")
-
-      {:error, :already_up} ->
-        CLI.info("The database for #{inspect(repo)} has already been created")
-
-      {:error, term} when is_binary(term) ->
-        CLI.error("The database for #{inspect(repo)} couldn't be created: #{term}")
-
-      {:error, term} ->
-        CLI.error("The database for #{inspect(repo)} couldn't be created: #{inspect(term)}")
+  def migrate() do
+    for repo <- repos() do
+      {:ok, _, _} = Ecto.Migrator.with_repo(repo, &Ecto.Migrator.run(&1, :up, all: true))
```

Comment:
Should we keep the logging `"Running migration for #{inspect(repo)}..."` (and also for rollbacks)? Doesn't look like [`Ecto.Migrator.with_repo/3`](https://github.com/elixir-ecto/ecto_sql/blob/v3.4.2/lib/ecto/migrator.ex#L124-L160) does any logging.

Reply:
Actually, migrate logs from the Migrator itself:
```
15:27:26.487 [info]  == Running 20180813131000 OMG.WatcherInfo.Repo.Migrations.CreateBlockTable.change/0 forward

15:27:26.492 [info]  create table blocks

15:27:26.497 [info]  == Migrated 20180813131000 in 0.0s

15:27:26.524 [info]  == Running 20180813131706 OMG.WatcherInfo.Repo.Migrations.CreateTransactionTable.change/0 forward

15:27:26.524 [info]  create table transactions

15:27:26.530 [info]  create index unq_transaction_blknum_txindex

15:27:26.532 [info]  == Migrated 20180813131706 in 0.0s

15:27:26.535 [info]  == Running 20180813133000 OMG.WatcherInfo.Repo.Migrations.CreateEtheventTable.change/0 forward

15:27:26.535 [info]  create table ethevents

15:27:26.539 [info]  == Migrated 20180813133000 in 0.0s

15:27:26.543 [info]  == Running 20180813143343 OMG.WatcherInfo.Repo.Migrations.CreateTxoutputTable.change/0 forward

15:27:26.543 [info]  create table txoutputs

15:27:26.555 [info]  == Migrated 20180813143343 in 0.0s

15:27:26.571 [info]  == Running 20190314105410 OMG.WatcherInfo.DB.Repo.Migrations.AlterTransactionsTableAddMetadataField.change/0 forward

15:27:26.571 [info]  alter table transactions

15:27:26.573 [info]  == Migrated 20190314105410 in 0.0s

15:27:26.581 [info]  == Running 20190315095855 OMG.WatcherInfo.DB.Repo.Migrations.AlterTransactionsTableAddPartialIndex.up/0 forward

15:27:26.581 [info]  execute "CREATE INDEX transactions_metadata_index ON transactions(metadata) WHERE metadata IS NOT NULL"

15:27:26.583 [info]  == Migrated 20190315095855 in 0.0s

15:27:26.586 [info]  == Running 20190408131000 OMG.WatcherInfo.Repo.Migrations.AddMissingIndicesToTxOuputs.change/0 forward

15:27:26.587 [info]  create index txoutputs_creating_txhash_spending_txhash_index

15:27:26.589 [info]  create index txoutputs_creating_deposit_index

15:27:26.590 [info]  create index txoutputs_spending_txhash_index

15:27:26.592 [info]  create index txoutputs_spending_exit_index

15:27:26.594 [info]  create index txoutputs_owner_index

15:27:26.597 [info]  == Migrated 20190408131000 in 0.0s

15:27:26.602 [info]  == Running 20190806111817 OMG.WatcherInfo.DB.Repo.Migrations.AlterTxOutputsTableAddRootchainTxnHashDepositAndExitColumns.up/0 forward

15:27:26.602 [info]  drop constraint txoutputs_creating_deposit_fkey from table txoutputs

15:27:26.604 [info]  drop constraint txoutputs_spending_exit_fkey from table txoutputs

15:27:26.605 [info]  drop constraint ethevents_pkey from table ethevents

15:27:26.608 [info]  drop table ethevents
```
is that enough?